.github/ISSUE_TEMPLATE/onboarding.yml

```diff
@@ -0,0 +1,61 @@
+name: Request for onboarding
+description: Ask Packit team to onboard my package(s)
+labels: onboarding
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello!
+        Don't have time to set up Packit? Or, do you want to see what it would look like on your package?
+
+        Fill out the following form and Packit team members will try to prepare a configuration file for you.
+
+        We will try our best and send the configuration file in the form of a pull request so you can see how it works.
+        If needed, we will tweak the configuration together with you to best meet your needs.
```

Comment:
Hmmm, maybe to describe what we are speaking about?

What about just describing what Packit can do for them instead in a simple paragraph?

Reply:
But I agree with @majamassarini that we should try to avoid as many details as possible because we are targeting people that don't want to go though the guide.

Reply:
>  we are targeting people that don't want to go through the guide.

or maybe don't know it exists as well? I think it would be nice "supporting" resource

Reply:
OK, I'll try to link it.